scaffold/tugboat/steps/2-update.sh.twig

```diff
@@ -7,7 +7,13 @@ set -eux
 echo "Updating..."
 
 composer install
-./vendor/bin/task {{ sync_command }}
+
+# Backwards compatibility for old sync command
+# If sync_command doesnt exists, run sync
+if ! ./vendor/bin/task {{ sync_command }}; then
```

Comment:
Will this always exist? I would have thought we'd resolve `sync_command` before rendering the template.